### PR TITLE
Make logging nicer, update beaker script

### PR DESCRIPTION
**Pull request by matt-gardner**

### CODE REVIEW

File: allennlp/training/trainer.py [hunk 1]
```diff
@@ -102,6 +103,8 @@ def __init__(self,
         if self._cuda_device >= 0:
             self._model = self._model.cuda(self._cuda_device)
 
+        self._log_interval = 10  # seconds
```
Comment (joelgrus) [inline allennlp/training/trainer.py]: is the thinking that we might want to allow people to override this? or should it be a module-level constant?

Comment (matt-gardner) [reply allennlp/training/trainer.py]: There was no such thinking =).  I don't have strong feelings about where this should go; if you do, I can move it.

Comment (joelgrus) [reply allennlp/training/trainer.py]: if we don't intend for users to modify it, I'd rather it be a module-level constant `_LOG_INTERVAL` than an instance variable. if we think they might want to, then leave it as is.

**Comment by joelgrus** [comment]
Thinking more, let's leave it this way

**Pull request merged**